crypto/ake/enclave/src/lib.rs

```diff
@@ -438,24 +440,43 @@ impl<EI: EnclaveIdentity> AkeEnclaveState<EI> {
     }
 
     /// Transforms an incoming client message, i.e. a message sent from a client
-    /// to the current enclave, into a list of outbound  messages for
-    /// other enclaves that serve as backends to the current enclave.
-    ///
-    ///                               / --> Backend Enclave 1
-    ///   Client -> Current Enclave ---> Backend Enclave 2
-    ///                              \ --> Backend Enclave N
```

Comment:
Can you re-add this in the new `reencrypt_sealed_message_for_backends` method? The visual aid promotes understanding more effectively IMO.